environment-dev.yml

```diff
@@ -37,6 +37,7 @@ dependencies:
   - sphinx-autodoc-typehints==1.11.1
   - sphinx_rtd_theme==0.5.2
   - pandas
+  - opencv-python
```

Comment:
U might need headless

Reply:
I've made it headless now, thanks for the pointer Sachi :)